src/Dnn.PersonaBar.Library/DTO/UserSettings.cs

```diff
@@ -27,6 +27,7 @@ namespace Dnn.PersonaBar.Library.DTO
     /// <summary>
     /// Persona Bar User Settings
     /// </summary>
+    [Obsolete("Deprecated in 9.2.1, use PersistSettings instead.")]
```

Comment:
This needs to be adjusted to add when the method will be removed? Which I believe would be v11? Right @mitchelsellers ?